recipe/meta.yaml

```diff
@@ -1,32 +1,28 @@
 {% set name = "Jinja2" %}
-{% set version = "3.1.4" %}
+{% set version = "3.1.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
-  sha256: 4a3aee7acbbe7303aede8e9648d13b8bf88a429282aa6122a993f0ac800cb369
-  patches:
-    - patches/0001-fix-test_package_zip_list-on-3.13.patch
+  sha256: 8fefff8dc3034e27bb80d67c671eb8a9bc424c0ef4c0826edbff304cceff43bb
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  skip: True  # [py<37]
+  skip: True  # [py<38]
 
 requirements:
-  build:
-    - patch # [unix]
-    - m2-patch # [win]
   host:
     - pip
     - python
     - flit-core <4
   run:
     - markupsafe >=2.0
     - python
+    - trio
```

Comment:
```suggestion
```
trio is not a run dependency: https://github.com/pallets/jinja/blob/3.1.5/pyproject.toml#L19
It should be added to test/requires instead.
See https://github.com/pallets/jinja/blob/3.1.5/requirements/tests.in

Reply:
Thanks!